_includes/i18n_dropdown_mobile.html

```diff
@@ -2,7 +2,7 @@
   <div class="sm-hide border-bottom border-blue">
     <div class="container cntnr-wide py1 px2 lg-px0 h5">
       <div class="center">
-        <span id="i18n-mobile-toggle" class="block text-decoration-none white fs-13p">
+        <span id="i18n-mobile-toggle" class="block text-decoration-none white fs-13p" role="button" tabindex="0">
```

Comment:
Yup! I had thought it was an error for two elements to have the same `tabindex` value, so if this is still valid, works for me! 💯